operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -219,6 +220,28 @@ public void stopNamespace(String ns) {
     }
   }
 
+  /**
+   * Report on currently suspended fibers. This is the first step toward diagnosing if we need special handling
+   * to kill or kick these fibers.
+   */
+  public void reportSuspendedFibers() {
+    if (LOGGER.isFineEnabled()) {
+      BiConsumer<String, FiberGate> consumer =
+          (namespace, gate) -> {
+            gate.getCurrentFibers().forEach(
+                (key, fiber) -> {
+                  Step lastIfSuspended = fiber.getLastIfSuspended();
```

Comment:
Consider using Optional.ofNullable(fiber.getLastIfSuspended()) rather than the separate get and if statement.